### ISSUE TITLE
Keyboard navigation for thumbnail filter does not work correctly

### ISSUE THREAD

**Issue by MatrixReverser** [MatrixReverser/VinylStudio#10]
When entering a keyword in the thumbnail filter, it can be set by pressing the tab key, but it does not work by pressing the enter key - as it does in the interpret filter.

Maybe the reason is that the following control (the data grid with thumbnails) cannot get the focus? Don't know, maybe.
